omnigibson/configs/fetch_behavior.yaml

```diff
@@ -1,5 +1,21 @@
 env:
+  action_timestep: 0.01666              # (float): environment executes action per action_timestep second
+  physics_timestep: 0.01666             # (float): physics timestep for physx
+  device: null                          # (None or str): specifies the device to be used if running on the gpu with torch backend
+  automatic_reset: false                # (bool): whether to automatic reset after an episode finishes
+  flatten_action_space: false           # (bool): whether to flatten the action space as a sinle 1D-array
+  flatten_obs_space: false              # (bool): whether the observation space should be flattened when generated
+  use_external_obs: false               # (bool): Whether to use external observations or not
   initial_pos_z_offset: 0.1
+  external_sensors:                     # (None or list): If specified, list of sensor configurations for external sensors to add. Should specify sensor "type" and any additional kwargs to instantiate the sensor. Each entry should be the kwargs passed to @create_sensor, in addition to position, orientation
```

Comment:
external sensor as in not binded to any robot?

Reply:
correct. actually, they could be implicitly bound to the robot, depending on the prim_path specified, so this `external_sensors` thing is quite flexible